insights/config/specs.py

```diff
@@ -220,6 +220,9 @@
     "modinfo"                   : CommandSpec("/usr/sbin/modinfo {module}", module=r"\S+"),
     "modprobe.conf"             : PatternSpec(r"etc/modprobe\.conf"),
     "modprobe.d"                : PatternSpec(r"etc/modprobe.d/.*\.conf"),
+    "mongod.conf"               : SimpleFileSpec("etc/mongod.conf"),
+    "mongod.conf_rh_mongodb26"  : SimpleFileSpec("etc/opt/rh/rh-mongodb26/mongod.conf"),
+    "mongodb.conf"              : SimpleFileSpec("etc/mongodb.conf"),
```

Comment:
1. As we discussed in today's meeting, better replace the '.' to '_'.
2. You can only declare one `spec` by using the `First` method to achieve your goal.
    Note: If you make sure there might be more specs on one host, please use `All` instead of `First`.
~~~
"mongod_conf": First([SimpleFileSpec("etc/mongod.conf"),SimpleFileSpec("etc/opt/rh/rh-mongodb26/mongod.conf"),SimpleFileSpec("etc/mongodb.conf")]),
~~~

Reply:
I will use All(...).